pronotepy/dataClasses.py

```diff
@@ -952,48 +954,124 @@ class Message(Object):
         the id of the message (used internally)
     author : str
         author of the message
-    recipients : list
-        Recipitents of the message. ! May be just ['# recipients'] !
     seen : bool
         if the message was seen
     date : datetime.datetime
         the date when the message was sent
     """
 
-    __slots__ = ["id", "author", "recipients", "seen", "date", "_client", "_listePM"]
+    __slots__ = ["id", "author", "seen", "date", "_client", "_listePM"]
 
     def __init__(self, client: _ClientBase, json_dict: dict) -> None:
         super().__init__(json_dict)
         self._client = client
-        self._listePM = json_dict["listePossessionsMessages"]["V"]
 
         self.id: str = self._resolver(str, "N")
         self.author: str = self._resolver(str, "public_gauche")
-        self.recipients: list = self._resolver(list, "listePublic")
-        self.seen: bool = self._resolver(bool, "lu")
+        self.seen: bool = self._resolver(bool, "lu", default=False)
         self.date: datetime.datetime = self._resolver(
-            lambda d: Util.datetime_parse(" ".join(d.split()[1:])), "libelleDate"
+            lambda d: Util.datetime_parse(d), "date", "V"
```

Comment:
instead of a lambda i think you can use `Util.datetime_parse` directly

Reply:
Not sure to understand what do you mezn😅

Reply:
Basically just
```suggestion
            Util.datetime_parse, "date", "V"
```

Reply:
Yes obviously I really don't know what I did 🤦‍♂️